### PR TITLE
spinner: added size progress

### PR DESCRIPTION
**Pull request by danielerez**
Added size progress indication for long steps

See: https://asciinema.org/a/583868

### CODE REVIEW

File: pkg/release/release.go [hunk 1]
```diff
@@ -204,34 +204,26 @@ func (r *release) mirrorImages(envConfig *config.EnvConfig, applianceConfig *con
...
+	cmd := fmt.Sprintf(ocMirrorAndUpload, absPath, registry.RegistryPort, tempDir)
+	logrus.Debugf("Fetching image from OCP release (%s)", cmd)
+
 	result, err := r.execute(r.executer, "", cmd)
 	logrus.Debugf("mirroring result: %s", result)
 	if err != nil {
 		return err
 	}
 
-	// remove oc mirror leftovers
```
Comment (nmagnezi) [inline pkg/release/release.go:225]: Why is this being removed?

Comment (nmagnezi) [reply pkg/release/release.go:225]: I mean, I get that this is related to what you did in above lines, but is it related to the PR $subject ?

Comment (danielerez) [reply pkg/release/release.go:225]: Yes, the change is required for monitoring the size of the mirror dir. I.e. it uses a relative path, so we can't use a syscall here to change the dir. Hence, had to change to logic a bit for displaying the progress correctly.

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**